compiler/plugins/target/AMD-AIE/aievec/VectorToVectorConversions.cpp

```diff
@@ -642,6 +642,7 @@ struct DetectNonCanonicalOpsPass
 };
 
 void buildCanonicalizeVectorForAIEVec(OpPassManager &pm) {
+
```

Comment:
Is this change on purpose?
```suggestion
```

Reply:
I don't know if this is a clang-format change or me being careless -- I'll commit the suggestion.